aws-s3-transfer-manager/src/operation/download/output.rs

```diff
@@ -42,35 +42,35 @@ pub struct ChunkOutput {
 // TODO: Do we want to expose something to yield multiple chunks in a single call, like
 // recv_many/collect, etc.? We can benchmark to see if we get a significant performance boost once
 // we have a better scheduler in place.
-impl Body {
-    /// Create a new empty body
+impl Output {
```

Comment:
**future**: We may want to have a conversion that allows you to just access the bytes off the stream (i.e. user only cares about the contents and not any of the intermediate metadata), we can wait though as it can be added in a backwards compatible way

```rs
fn into_body(self) -> Body
```

Reply:
Discussed offline, reverted the name change.